examples/powertools-examples-cloudformation/infra/sam/template.yaml

```diff
@@ -0,0 +1,54 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  powertools-examples-cloudformation
+
+  Sample SAM Template for powertools-examples-cloudformation
+
+Globals:
+  Function:
+    Timeout: 20
+    MemorySize: 128
```

Comment:
Memory size is defined and then overridden in the function. Maybe remove the global value.

Reply:
removing it